chapter_07_aggregate.asciidoc

```diff
@@ -850,3 +852,38 @@ a|
 
 * Dealing with eventual consistency between aggregates can be complex.
 |===
+
+Part one Recap
```

Comment:
options, neither of which i'm particularly wedded to: 

- keep this at end of chapter 7
- give it its own chapter
- put it in the part 2 intro

i don't have strong enough feelings to make a change, but just in case one of these sparks interest...